styles/xrpl.scss

```
@@ -69,6 +69,7 @@ $line-height-base: 1.5;
 @import "_video.scss";
 //@import "_top-banner.scss";
 @import "_toml-checker.scss";
+@import "_docs.scss";
```

If you rename this file as I recommended above, don't forget to change it here, too.

```suggestion
@import "_docs-home.scss";
```